mmv1/products/gkeonprem/VmwareCluster.yaml

```diff
@@ -0,0 +1,662 @@
+              NodePort for ingress service's http. The ingress service in the admin
+              cluster is implemented as a Service of type NodePort (ex. 32527).
+            default_from_api: true
+          - !ruby/object:Api::Type::Integer
+            name: "ingressHttpsNodePort"
+            description: |
+              NodePort for ingress service's https. The ingress service in the admin
+              cluster is implemented as a Service of type NodePort (ex. 30139).
+            default_from_api: true
+          - !ruby/object:Api::Type::Integer
+            name: "controlPlaneNodePort"
+            description: |
+              NodePort for control plane service. The Kubernetes API server in the admin
+              cluster is implemented as a Service of type NodePort (ex. 30968).
+            default_from_api: true
+          - !ruby/object:Api::Type::Integer
+            name: "konnectivityServerNodePort"
+            description: |
+              NodePort for konnectivity server service running as a sidecar in each
+              kube-apiserver pod (ex. 30564).
+            default_from_api: true
+      - !ruby/object:Api::Type::NestedObject
+        name: "metalLbConfig"
+        description: Configuration for MetalLB typed load balancers.
+        exactly_one_of:
+          - f5_config
+          - manual_lb_config
+          - metal_lb_config
+        properties:
+          - !ruby/object:Api::Type::Array
+            name: "addressPools"
+            description: |
+              AddressPools is a list of non-overlapping IP pools used by load balancer
+              typed services. All addresses must be routable to load balancer nodes.
+              IngressVIP must be included in the pools.
+            item_type: !ruby/object:Api::Type::NestedObject
+              properties:
+                - !ruby/object:Api::Type::String
+                  name: "pool"
+                  description: |
+                    The name of the address pool.
+                - !ruby/object:Api::Type::Array
+                  name: "addresses"
+                  item_type: Api::Type::String
+                  description: |
+                    The addresses that are part of this pool. Each address 
+                    must be either in the CIDR form (1.2.3.0/24) or range 
+                    form (1.2.3.1-1.2.3.5).
+                - !ruby/object:Api::Type::Boolean
+                  name: "avoidBuggyIps"
+                  description: |
+                    If true, avoid using IPs ending in .0 or .255.
+                    This avoids buggy consumer devices mistakenly dropping IPv4 traffic for
+                    those special IP addresses.
+                  default_from_api: true
+                - !ruby/object:Api::Type::Boolean
+                  name: "manualAssign"
+                  description: |
+                    If true, prevent IP addresses from being automatically assigned.
+                  default_from_api: true
```

Comment:
Does this default to `true` in the API? If so, we likely want to use a clientside default + apply `custom_flatten: "templates/terraform/custom_flatten/default_if_empty.erb"`

Reply:
This is default to fale. Thanks for confirmating

Reply:
```suggestion
```

We should be able to remove `default_from_api: true` in that case!